app/services/suite_selector.rb

```diff
@@ -34,7 +36,7 @@ def select
 
   private
 
-  attr_accessor :errors
+  attr_writer :errors
```

Comment:
I believe that this overwrites the public reader (if I'm remembering correctly).